### DISCUSSION TITLE
How do I achieved nested routing with creating absolute path?

### DISCUSSION THREAD

**Discussion by abbbdab** [remix-run/remix#9770] [Q&A]
So I have this folder structure in a project. I want to have all of links (iterated over an array) on the outlet level. Currently I have to provide the full path in order to navigate, whereas I want to achieve same behaviour if each navlink was present on its respective route. 
![Screenshot 2024-07-18 at 6 25 53 pm](https://github.com/user-attachments/assets/4c96d51e-ab5f-4edc-a431-a4f738ab2839)

**Comment by suhaotian** [comment]
For nested routes, it is better to use [manual route configuration](https://remix.run/docs/en/main/discussion/routes#manual-route-configuration). (For code consistency and folder organization in medium or large project, I prefer to use this method for all routes.)

**Reply by abbbdab** [reply]
@suhaotian but can I use it in tandem with traditional routing? Like some routes with manual config and the others with file based routing?

**Reply by suhaotian** [reply]
Yes, you can
